### DISCUSSION TITLE
Problem with homebrew in Mac M1

### DISCUSSION THREAD

**Discussion by dectun** [Homebrew/discussions#3185] [Getting started]
Hello. I try many ways, but I can`t build MVE, and the probably wrong stay in homebrew. I reinstalled MVE. and have the same problem with de png libraries. 

You can see that all i did in this link;
(https://github.com/simonfuhrmann/mve/issues/559#issue-1204960021)

When is try with brew doctor, it say me is ok. When I try reinstall libraries with brew, it say me that all is installed. But when I try build MVE, "it say me: fatal error: "png.h" not found".
![Captura de Pantalla 2022-04-15 a las 1 08 17](https://user-images.githubusercontent.com/103385292/163491608-40e2b0d7-630c-448c-b7ff-6b657b7f9ffc.png)

**Comment by SMillerDev** [comment]
Since this is not a problem with homebrew but a problem with usage of homebrew packages there's not much help I can give you. I pointed you at the M1 specific instructions but that's all I can really find about mve.

We don't know how to fix all software in the world, just most of the software we package.

**Comment by carlocab** [comment]
Try doing
```sh
export CFLAGS="-I$(brew --prefix)/include"
```
before running `make`.

**Reply by dectun** [reply]
I tried, but give me the same error.

I will try to uninstall all, and reinstall. 

Very much thanks for all help that you give me. And so sorry for all problems that I say you.
